package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "emoji-picker-react",
-  "version": "3.4.12",
+  "version": "3.4.11",
```

Comment:
I just realized that the version is now lower. Since this is a new feature, it should be a minor change - so `3.5.0` would be a better fit.